wordpress-mobile/WordPress-Android
# Add settings to remove location from images

Fixes #8557 
FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1059

Adds a `Remove location` setting to App settings. It is unchecked by default. When it is checked, the geoinformation in the Exif information on every image should be stripped when it's being uploaded to the server.  

To test:
* Upload a photo (for example from the post editor)
* Download the photo from a browser
* Check the Exif information
* All the Geoinformation should be set to 0

Update release notes:
